examples/playwright-sandbox/README.md

```
@@ -0,0 +1,105 @@
+
+### Check Sandbox status
+
+```bash
+kubectl get sandbox playwright-sandbox
+kubectl get pods -l app=playwright-sandbox
+POD=$(kubectl get pods -l app=playwright-sandbox -o jsonpath='{.items[0].metadata.name}')
+kubectl logs ${POD} -c playwright
+```
+
+You should see output similar to:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)